src/build/buildExecutor.ts

```diff
@@ -193,4 +237,11 @@ export class BuildExecutor {
         }
         return cmdWithParameters;
     }
+
+    private getServeExecArgs(input: BuildInput, isPublicUser: boolean): string[] {

```

Comment:
Can we add this to the `getBuildParameters` since is also a part of build parameters